### PR TITLE
[4.0] Missing constant in com_installer.sys.ini

### PR DESCRIPTION
**Pull request by Kostelano**
### Summary of Changes
Missing constant in com_installer.sys.ini. Can be seen by adding a menu item to the CONTROL PANEL menu.

Screenshot_1

### CODE REVIEW

File: administrator/language/en-GB/com_installer.sys.ini [hunk 1]
```diff
@@ -19,6 +19,7 @@ COM_INSTALLER_MANAGE_VIEW_DEFAULT_TITLE="Manage Extensions"
 COM_INSTALLER_UPDATE_VIEW_DEFAULT_DESC="Find and Install updates for the installed extensions."
 COM_INSTALLER_UPDATE_VIEW_DEFAULT_TITLE="Update Extensions"
 COM_INSTALLER_UPDATE_VIEW_DISPLAY_BADGE="Display Badge"
+COM_INSTALLER_UPDATESITES_VIEW_DEFAULT_DESC="Manage update sites for various installed extensions."
```
Comment (brianteeman) [inline administrator/language/en-GB/com_installer.sys.ini]: not just for extensions - also for core. I suggest

COM_INSTALLER_UPDATESITES_VIEW_DEFAULT_DESC="Displays a list of all the update sites that will be checked."

**Comment by Kostelano** [comment]
Updated, thanks. I just took the constant from Joomla 3.

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on 5dfaf104a04ad5553b7ee1ae845490e8ebfad68f<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/34190">issues.joomla.org/tracker/joomla-cms/34190</a>.</sub>

**Comment by ghost** [comment]
I have tested this item :white_check_mark: successfully on 5dfaf104a04ad5553b7ee1ae845490e8ebfad68f<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/34190">issues.joomla.org/tracker/joomla-cms/34190</a>.</sub>

**Comment by alikon** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/34190">issues.joomla.org/tracker/joomla-cms/34190</a>.</sub>

**Comment by chmst** [comment]
Thanks!

**Pull request merged**